packages/wrangler/src/index.tsx

```diff
@@ -605,6 +606,22 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
+		"workers",
```

Comment:
Do we actually want to call these "workers"?
Are we not leaning towards "services"?

Reply:
What about something like `list`? (We could even make it slightly more generic and accept things like `wrangler list kv` to list kv namespaces, `wrangler list workers/services`, wrangler list r2` etc...).

Reply:
We already have a command called `deployments` why not just add it to that command as a flag `--services-list` or something? I am also not against the `list` command